scss/_patterns_card.scss

```diff
@@ -30,7 +30,7 @@
 
 %p-card--highlighted {
   @extend %p-card;
-  box-shadow: 0 1px 5px 1px transparentize($color-dark, .8);
+  box-shadow: 0 1px 5px 0 transparentize($color-dark, .8);
```

Comment:
Shouldn't this be 0 1px 2px 0 according to #1614?

Reply:
Good catch - thank you!